src/Helper/AbstractHelper.php

```diff
@@ -18,7 +18,7 @@ abstract class AbstractHelper implements HelperInterface
     /**
      * Set the View object
      *
-     * @return $this
+     * @return self
```

Comment:
Psalm is happier with this

Reply:
Seems to work fine? :thinking: 

https://psalm.dev/r/29584fecc0

Reply:
It's to avoid generating more errors in other implementors because of the trait:
https://psalm.dev/r/331a66aa21

Reply:
Makes sense.